### PR TITLE
Add Identity LinearOperator

### PR DESCRIPTION
**Pull request by fzimmermann89**
Add a LinearOperator Identity, i.e. Identity with a single input tensor.

(This might be extended for multiple inputs)

### CODE REVIEW

**Comment by fzimmermann89** [comment]
This allows us to write for example
H=A.H@A+lam*Identity()
In CG sense

**Comment by fzimmermann89** [comment]
We could also add a identity instance to the init/make it static.

This would allow us to write just "Identity" or "I" instead of IdentityOp().
This is possible if all identity operator instances do exactly the same thing..

File: src/mrpro/operators/IdentityOp.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    A Linear Operator that returns a single input unchanged.
+    """
+
+    def __init__(self) -> None:
+        """Initialize Identity Operator."""
+        super().__init__()
+
+    def forward(self, x: torch.Tensor) -> tuple[torch.Tensor]:
+        """Identity of input `x`.
```
Comment (ckolbPTB) [inline src/mrpro/operators/IdentityOp.py]: The ` before and after x will lead to x being printed slightly italic in the docu. Is this what you want here or did you want it to be a literal? 

see e.g. https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html

File: src/mrpro/operators/IdentityOp.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
```
Comment (ckolbPTB) [inline src/mrpro/operators/IdentityOp.py:25]: Maybe add 
```
Parameters
----------
```
for sake of completeness

**Comment by ckolbPTB** [comment]
I don't think we should change the EinsumOp in this PR even if it is just doc string formatting.

**Comment by fzimmermann89** [comment]
I am impartial about it and can remove that change.

**Review by ckolbPTB — APPROVED** [review]

**Pull request merged**